easybuild/easyconfigs/n/NLopt/NLopt-2.6.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'CMakeMake'
+
+name = 'NLopt'
+version = '2.6.1'
+
+homepage = 'http://ab-initio.mit.edu/wiki/index.php/NLopt'
+description = """ NLopt is a free/open-source library for nonlinear optimization, 
+ providing a common interface for a number of different free optimization routines 
+ available online as well as original implementations of various other algorithms. """
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/stevengj/nlopt/archive']
+sources = [{
+    'download_filename': 'v%(version)s.tar.gz',
+    'filename': SOURCE_TAR_GZ
```

Comment:
@fizwit Any specific reason for renaming the source tarball after download? Why not go simply with

```python
sources = ['v%(version)s.tar.gz']

Reply:
@fizwit I don't see the point in renaming the source tarball after downloading?

Reply:
I've made this change myself, I see no reason for renaming this after download (we don't do this in many other easyconfigs either).